include/zephyr/net/wifi_mgmt.h

```diff
@@ -263,6 +266,15 @@ struct wifi_reg_domain {
 	uint8_t country_code[WIFI_COUNTRY_CODE_LEN];
 };
 
+enum wifi_twt_sleep_status {
+	WIFI_TWT_STATUS_SLEEP = 0,
+	WIFI_TWT_STATUS_AWAKE,
+};
+
+struct wifi_twt_state {
+	enum wifi_twt_sleep_status sleep_status;
```

Comment:
are you expecting more fields in the future? If not we can get rid of the structure and just a boolean/int?